index.html

```diff
@@ -0,0 +1,258 @@
+              alt="languages"
+            />
+            <h2 class="about-desc-font">Languages</h2>
+            <ul class="about-section-detail tags">
+              <li class="lists">RubyOnRails</li>
+              <li class="lists">HTML</li>
+              <li class="lists">CSS</li>
+              <li class="lists">JavaScript</li>
+            </ul>
+          </div>
+          <div class="about-desc">
+            <img
+              class="about-icon"
+              src="./images/icons/about_icon2.svg"
+              alt="framewords"
+            />
+            <h2 class="about-desc-font">Frameworks</h2>
+            <ul class="about-section-detail tags">
+              <li class="lists">RubyOnRails</li>
+              <li class="lists">Bootstrap</li>
+              <li class="lists">RSpec</li>
+              <li class="lists">Selenium</li>
+              <li class="lists">Selenium</li>
+            </ul>
+          </div>
+          <div class="about-desc">
+            <img
+              class="about-icon"
+              src="./images/icons/about_icon3.svg"
+              alt="Skills"
+            />
+            <h2 class="about-desc-font">Skills</h2>
+            <ul class="about-section-detail tags">
+              <li class="lists">GitHub</li>
+              <li class="lists">CodePen</li>
+              <li class="lists">GitLab</li>
+              <li class="lists">Terminal</li>
+            </ul>
+          </div>
+        </div>
+      </section>
+      <footer id="contact-message">
+        <div id="footer-top">
+          <h2 id="footer-text" class="color-base">
+            I'm always interested in hearing about new projects, so if you'd like
+            to chat please get in touch.
+          </h2>
+          <form id="form" action="https://formspree.io/f/mrgjqpap" method="POST">
+            <label hidden>Name</label>
+            <input
+              class="form-fields"
+              type="text"
+              name="name"
+              id="name"
+              maxlength="30"
+              placeholder="Full Name"
+              required
+            />
+            <label hidden>Email Address</label>
+            <input
+              class="form-fields"
+              type="email"
+              name="Email Address"
+              id="email-address"
+              placeholder="Email Address"
+              required
+            />
+            <small id="email-invalidation-message"></small>
+            <label hidden>Subject</label>
+            <textarea
+              class="form-fields"
+              name="message"
+              id="message"
+              maxlength="500"
+              required
+            >
+      Describe you subject here...</textarea
+            >
+            <button
+              type="submit"
+              id="get-in-touch"
+              class="primary-button button-states"
+            >
+              Get in touch
+            </button>
```

Comment:
[OPTIONAL] 
![image](https://user-images.githubusercontent.com/61361037/211077266-f5f1357d-1361-4e95-87b7-2790dd55386a.png)
The shadow created by hovering over the buttons is too much. You can reduce the shadow a little bit to make it beautiful. This much shadow could look beautiful if there is some transition effect that could smooth the transition out.